cl/settings/django.py

```diff
@@ -182,11 +182,21 @@
     "cl.stats",
     "cl.users",
     "cl.visualizations",
+    "tailwind",
 ]
 
 if DEVELOPMENT:
     INSTALLED_APPS.append("django_extensions")
-    MIDDLEWARE.append("debug_toolbar.middleware.DebugToolbarMiddleware")
+
```

Comment:
Note this is updated to mirror the setup [here](https://github.com/freelawproject/bigcases2/blob/main/bc/settings/django.py) in BigCases